### PR TITLE
Add internal-apis client

### PR DESCRIPTION
**Pull request by sayplastic**

### CODE REVIEW

File: extras/lbryinc/client.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+	client := &http.Client{Timeout: timeout}
+	r, err := client.Do(req)
+	if err != nil {
+		return body, err
+	}
+	defer r.Body.Close()
+
+	body, err = ioutil.ReadAll(r.Body)
+	return body, err
```
Comment (nikooo777) [inline extras/lbryinc/client.go]: you can shorten this to
`return ioutil.ReadAll(r.Body)`

File: extras/lbryinc/client.go [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+// Client stores data about internal-apis call it is about to make.
+type Client struct {
+	ServerAddress string
+	AuthToken     string
+	Logger        *logrus.Logger
+}
+
+// APIResponse reflects internal-apis JSON response format.
+type APIResponse struct {
```
[the remark this answers is not in the document]
Comment (sayplastic) [reply extras/lbryinc/client.go]: the fields do not match unfortunately
